### PR TITLE
fix: remove "Ideas for new ideas"  section from Report List

### LINKED ISSUE #1194: remove broken feature
**Summary:** 

Remove the "Ideas For New Features"  section  from Report Type List.

**Screenshots:** 

Before:

https://github.com/OneBusAway/onebusaway-android/assets/114914556/2720c434-0206-48ce-a36b-c97353cf2e51

After:

### PR DESCRIPTION
**Pull request by sebesti0n**
Fixed #1194 

Preview:

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `gradlew connectedObaGoogleDebugAndroidTest` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them for the initial submission of the pull request.  When addressing comments on a pull request, please push a new commit per comment when possible (reviewers will squash and merge using GitHub merge tool)

### CODE REVIEW

File: onebusaway-android/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -585,18 +585,16 @@
     <string name="rt_infrastructure_problem">Segnala problema con la fermata</string>
     <string name="rt_customer_service">Contatta il Servizio clienti</string>
     <string name="rt_stop_problem">Segnala problema con la fermata</string>
-    <string name="rt_arrival_problem">Segnala problema con l\'orario d\'arrivo</string>
+    <string name="rt_arrival_problem">Segnala problema con l\'orario d\'arrivo</string>List
```
Comment (aaronbrethorst) [inline onebusaway-android/src/main/res/values-it/strings.xml:588]: There's a bit of invalid XML hanging out at the end of this line.

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
works as expected. However, in addition to the comment in the values-it XML file, there's still a couple references to Ideascale in the codebase. Please do a find in files, locate them, and delete them.

File: onebusaway-android/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (sebesti0n) [reply onebusaway-android/src/main/res/values-it/strings.xml:588]: Okay i, remove it.

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
I still see references to Ideascale in the codebase

**Comment by sebesti0n** [comment]
> I still see references to Ideascale in the codebase

Done sir

**Review by aaronbrethorst — APPROVED** [review]
looks good, thanks! 👍

**Pull request merged**